src/bci_build/templates.py

```diff
@@ -91,6 +91,8 @@
 <!-- {{ INFOHEADER }}-->
 <!-- OBS-AddTag: {% for tag in image.build_tags -%} {{ tag }} {% endfor -%}-->
 <!-- OBS-Imagerepo: obsrepositories:/ -->
+{%- if image.os_version.is_sle15 and image.name == "base" %}
+<!-- OBS-IgnorePackage: rpm -->{% endif %}
```

Comment:
Can we please not have hacks like this in the template? Image specific settings belong into the image and not into the general purpose template imho

Reply:
added a separate kiwi_ignore_packages flag for this.